src/assets/styles/footer.scss

```diff
@@ -0,0 +1,89 @@
+.footer {
+  display: inline-block;
+  background: $light-gray;
+  height: 10em;
+  width: 100%;
+
+  .footer-buttons {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-wrap: wrap;
+  }
+
+  a.btn {
+    margin: 1.5em 0.6em 0.3em 0.6em;
+    color: white;
+    text-decoration: none;
+    flex-grow: 1;
+    font-size: 1em;
+    text-align: center;
+    position: relative;
+    min-width: calc(50% - 2.5em);
+  }
+
+  .privacy-policy {
+    margin: 0 1em;
+    &:link, &:visited, &:hover {
```

Comment:
Oops! My bad on the suggestion before, these really need to be `a:link, a:visited, a:hover` to work -- I missed that it is an anchor _inside_ the `.privacy-policy` selector.